<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>CryptologyTool</class>
  <widget class="QMainWindow" name="CryptologyTool">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>800</width>
        <height>600</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>CryptologyTool</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
          <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
              <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
              <attribute name="title">
                <string>对称加密</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                  <widget class="QLabel" name="label_5">
                    <property name="text">
                      <string>算法</string>
                    </property>
                  </widget>
                </item>
                <item row="1" column="0">
                  <widget class="QLabel" name="label">
                    <property name="text">
                      <string>明文</string>
                    </property>
                  </widget>
                </item>
                <item row="1" column="1">
                  <widget class="QLineEdit" name="lineEdit_PlainText" />
                </item>
                <item row="1" column="2">
                  <widget class="QLabel" name="label_6">
                    <property name="text">
                      <string>初始化向量</string>
                    </property>
                  </widget>
                </item>
                <item row="1" column="3">
                  <widget class="QLineEdit" name="lineEdit_EncryptionKey_2">
                    <property name="enabled">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="1" column="4">
                  <widget class="QLabel" name="label_3">
                    <property name="text">
                      <string>加密密钥</string>
                    </property>
                  </widget>
                </item>
                <item row="1" column="5" colspan="2">
                  <widget class="QLineEdit" name="lineEdit_EncryptionKey" />
                </item>
                <item row="1" column="7">
                  <widget class="QPushButton" name="pushButton_Encryption">
                    <property name="text">
                      <string>加密</string>
                    </property>
                  </widget>
                </item>
                <item row="2" column="0">
                  <widget class="QLabel" name="label_2">
                    <property name="text">
                      <string>密文</string>
                    </property>
                  </widget>
                </item>
                <item row="2" column="1">
                  <widget class="QLineEdit" name="lineEdit_Ciphertext" />
                </item>
                <item row="2" column="2">
                  <widget class="QLabel" name="label_7">
                    <property name="text">
                      <string>初始化向量</string>
                    </property>
                  </widget>
                </item>
                <item row="2" column="3">
                  <widget class="QLineEdit" name="lineEdit_EncryptionKey_3" />
                </item>
                <item row="2" column="4" colspan="2">
                  <widget class="QLabel" name="label_4">
                    <property name="text">
                      <string>解密密钥</string>
                    </property>
                  </widget>
                </item>
                <item row="2" column="6">
                  <widget class="QLineEdit" name="lineEdit_DecryptionKey" />
                </item>
                <item row="2" column="7">
                  <widget class="QPushButton" name="pushButton_Decryption">
                    <property name="text">
                      <string>解密</string>
                    </property>
                  </widget>
                </item>
                <item row="3" column="0" colspan="8">
                  <widget class="QTextBrowser" name="textBrowser" />
                </item>
                <item row="0" column="1" colspan="7">
                  <widget class="QComboBox" name="comboBox" />
                </item>
              </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
              <attribute name="title">
                <string>非对称加密</string>
              </attribute>
            </widget>
          </widget>
        </item>
      </layout>
    </widget>
    <widget class="QStatusBar" name="statusbar" />
  </widget>
  <resources />
  <connections />
</ui>
